packages/base/src/3dview/mainview.tsx

```diff
@@ -1075,6 +1054,8 @@ export class MainView extends React.Component<IProps, IStates> {
     } else {
       this._refLength = null;
     }
+
+    this._updateCamera();
```

Comment:
If possible, we shouldn't reset the camera entirely here (note we're already have some code in this method that updates some camera parameters). 
Instead, we should only set the new `near` and `far` if it's an orthographic camera.